pkg/app/piped/cloudprovider/kubernetes/diff.go

```diff
@@ -113,6 +139,56 @@ func (r *DiffListResult) DiffString() string {
 	return b.String()
 }
 
+func diffByCommand(old, new Manifest) ([]byte, error) {
+	oldBytes, err := old.YamlBytes()
+	if err != nil {
+		return nil, err
+	}
+
+	newBytes, err := new.YamlBytes()
+	if err != nil {
+		return nil, err
+	}
+
+	oldFile, err := os.CreateTemp("", "old")
+	if err != nil {
+		return nil, err
+	}
+	defer os.Remove(oldFile.Name())
+	if _, err := oldFile.Write(oldBytes); err != nil {
+		return nil, err
+	}
+
+	newFile, err := os.CreateTemp("", "new")
+	if err != nil {
+		return nil, err
+	}
+	defer os.Remove(newFile.Name())
+	if _, err := newFile.Write(newBytes); err != nil {
+		return nil, err
+	}
+
+	cmd := "diff"
+	data, err := exec.Command(cmd, "-u", "-N", oldFile.Name(), newFile.Name()).CombinedOutput()
```

Comment:
nits, how about using standard go diff. The implementation uses the os diff command just like what we do here but we could omit the exec call here 👀 
ref: https://pkg.go.dev/cmd/internal/diff#Diff

Reply:
Because it is placing in an internal package.

Reply:
Ah, my bad 🙆‍♀️